BallisticTables.json

```diff
@@ -0,0 +1,26 @@
+{"82mm_a3": [
+    {"charge": 0, "table": [
```

Comment:
I was thinking about that, and it looks doable, but will we then need a key that states what the minimum and maximum ranges available for the charge are so we don't try to find a value that's not present, or can we solve that in code?

What happens when you try to read a key that doesn't exist? Exception handling?